e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/security/accesscontrol/deleteput/ResourceDto.java

```diff
@@ -0,0 +1,17 @@
+package com.foo.rest.examples.spring.security.accesscontrol.deleteput;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
```

Comment:
`@ApiModel` and `@ApiModelProperty` are not needed in E2E, as they only impact text in generate schema

Reply:
Hi Andrea, thanks for your comments. I removed them as suggested.